dataplane/src/nat/mod.rs

```diff
@@ -393,10 +393,11 @@ impl<Buf: PacketBufferMut> NetworkFunction<Buf> for Nat {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::pipeline::test_utils::build_test_ipv4_packet;
     use iptrie::Ipv4Prefix;
     use net::buffer::TestBuffer;
     use net::headers::TryIpv4;
+    #[cfg(test)]
+    use net::packet::test_utils::build_test_ipv4_packet;
```

Comment:
This is already under `#[cfg(test)]`, I don't think we need to add a second `#[cfg(test)]`?

Reply:
true, it's not needed. Will remove